misk-zookeeper/src/main/kotlin/misk/clustering/zookeeper/AtomicEnum.kt

```diff
@@ -0,0 +1,27 @@
+package misk.clustering.zookeeper
+
+import java.util.concurrent.atomic.AtomicInteger
+
+/** An [AtomicEnum] is an enum that can be updated atomically, including down an atomic compare and set */
```

Comment:
not quite following the second part of the sentence: "including down an atomic compare and set"

Reply:
(maybe rm down?)

Reply:
oops, fixed